Source/Core/VideoCommon/FreeLookCamera.cpp

```diff
@@ -200,34 +200,35 @@ class OrbitalController final : public CameraControllerInput
   }
 
 private:
-  float m_distance = 0;
+  static constexpr float MIN_DISTANCE = 0.0f;
+  float m_distance = MIN_DISTANCE;
   Common::Vec3 m_rotation = Common::Vec3{};
   Common::Quaternion m_rotate_quat = Common::Quaternion::Identity();
 };
 }  // namespace
 
 Common::Vec2 CameraControllerInput::GetFieldOfView() const
```

Comment:
Perhaps this function should also be named `GetFieldOfViewMultiplier` for consistency.